digitalpublicgoods/digit-public-finance-management.json

```diff
@@ -0,0 +1,96 @@
+{
+  "name": "DIGIT Public Finance Management",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://ifix.digit.org/"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://ifix.digit.org/platform/strategy-and-approach/functional-specifications",
+      "https://ifix.digit.org/platform/strategy-and-approach/technical-specification"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "Yes",
+    "checkNonPIIAccessMechanism": "Yes",
+    "nonPIIAccessMechanism": "Dashboards and data dump downloads from dashboards in XLS format."
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Yes",
+    "privacyComplianceList": [
+      "Information Technology Act, 2000 (as interpreted by the Supreme Court of India in the Puttaswamy - 1 judgment)"
+    ],
+    "adherenceSteps": [
+      "Steps taken for adherence are in compliance with the Information Technology Act, 2000 (as interpreted by the Supreme Court of India in the Puttaswamy - 1 judgment) Steps are -: Purposeful collection, Legitimate interest in data collection &  Data minimization."
+    ]
+  },
+  "standards": {
+    "supportStandards": "Yes",
+    "standardsList": [
+      "Open API Specifications",
+      "JSON",
+      "REST"
+    ],
+    "evidenceStandardSupport": [
+      "https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/egovernments/iFix-Dev/develop/domain-services/ifix-master-data-service/ifix-master-data-service-1.0.0.yaml",
+      "https://github.com/egovernments/iFix-Dev/blob/master/domain-services/fiscal-event-service/fiscal-event-service-1.0.0.yaml",
+      "https://github.com/egovernments/iFix-Dev/blob/master/domain-services/fiscal-event-post-processor/fiscal-event-post-processor-1.0.0.yaml",
+      "https://github.com/egovernments/iFix-Dev/blob/master/domain-services/ifix-department-entity-service/ifix-department-entity-service-1.0.0.yaml"
+    ],
+    "implementBestPractices": "Yes",
+    "bestPracticesList": [
+      "Interoperable, Standards based, Minimalism, Open Source, Unbundled, Secure & Reliable, Scalable, Ease of use, Inclusive"
```

Comment:
Please ask submitter to clarify how DIGIT iFix adheres to the following best practices listed. Thanks.

Reply:
- iFix uses open source technologies throughout the tech stack. For more details about the tech stack, please see this link.
- iFIX leverages JSON for data exchange via APIs. APIs are defined using Open API
specifications. All services are documented here.
iFix is built using microservices architecture. All services are “unbundled” independent, minimal and can be used across contexts.
- Since iFix is built on microservices, each component can be evolved and
independently scaled which makes the platform as a whole scalable. iFix can be
implemented across commercial and private clouds irrespective of the provider.
- All the APIs are well documented and in the open for everyone to consume.
Extensive documentation ensures that the platform is easy to use.